CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ lxqt_translate_ts(QM_FILES
     ${UIS}
     INSTALL_DIR
     "${LXQT_TRANSLATIONS_DIR}/${PROJECT_NAME}"
+    PULL_TRANSLATIONS ${PULL_TRANSLATIONS}
```

Comment:
Why not stating the variables PULL_TRANSLATIONS & CLEAN_TRANSLATIONS as `option`s?